### DISCUSSION TITLE
Release Dates

### DISCUSSION THREAD

**Discussion by brianteeman** [joomla/joomla-cms#31713] [General] [answered]
Please no release for joomla over the holiday period. Like it or not a vast percentage of the userbase will not be working between the 24th and the 2nd.

**Comment by PhilETaylor** [comment]
<img width="349" alt="Screenshot 2020-12-18 at 21 35 27" src="https://user-images.githubusercontent.com/400092/102663548-f9565e00-4178-11eb-8296-4f77f5ae0bef.png">
<img width="308" alt="Screenshot 2020-12-18 at 21 19 42" src="https://user-images.githubusercontent.com/400092/102662446-c4e1a280-4176-11eb-90f8-8a6a9c4a655f.png">

**Comment by richard67** [comment]
Well, 29th is not a public holiday in most countries. But I agree that many people normally are in holiday at this day. This year it might be different due to COVID travelling and meeting restrictions, but of course we should not rely on that. If I could chose, my favourite release day would be Tuesday 22nd. But I have no power to decide.

**Reply by richard67** [reply]
... or maybe later in January, that would leave more time for proper testing and bug fixing.

**Reply by brianteeman** [reply]
It's not just about the single day of the release.

We know from multiple experiences that a patch release can sometimes have greater consequences than expected, especially with any security fixes which by their nature do not get as extensively tested. Many times we have had to release a version almost immediately to fix something that was unforeseen. 

Are people going to be around to work on that?  Are people going to be happy that their site is down/broken when they should be relaxing with their family? Will they want to be updating their site again on New Years Eve.

All the reasons for releasing on a Tuesday still apply. But over the holiday period they are enhanced and exaggerated.

**Comment by HLeithner** [comment]
Since people are unhappy with the release date we postpone the release to 12.01.2021

**Reply by PhilETaylor** [reply]
People are unhappy because it was a silly decision in the first place. Don't blame people for being unhappy for the justification of the  move. Blame the fact that it was an silly decision from the start :) Thank you for communicating the new date with plenty of time for people to make new plans.

**Comment by brianteeman** [comment, marked as answer]
Thank you for listening to the voices of experience.


PS how do we close this?

**Reply by HLeithner** [reply]
I changed it to Q & A section now you can accept the Answer... I think github have some points to improve for discussions

**Reply by richard67** [reply]
Or we could create a new category for closed discussions. What do you think? What's the better way?

**Comment by brianteeman** [comment]
Just saw today that it has actually been documented for over two years so this issue should never have arisen if the documented release procedures were followed

>  Note that when scheduling releases, consideration should be given to holidays around the world and the impact of issuing a release near those periods considered

Source: [Release procedure and checklist
](https://docs.joomla.org/Joomla:Release_procedure_and_checklist#Preparing_To_Release)
